### DISCUSSION TITLE
Change notification text in Slack

### DISCUSSION THREAD

**Discussion by alexr3** [DefectDojo/django-DefectDojo#6226] [Q&A] [answered]
Hello!

How to change notification text after integrating with Slack?

I'm interested in the part of the notification that specifies the path to the report:
**They can be viewed here: http://localhost:8080/test/1**

How to change http://localhost:8080 to my url?

Installation type DD - Docker Compose

**Comment by alexr3** [comment]
Somebody knows?

**Comment by Juu** [comment, marked as answer]
There is an environment variable `DD_SITE_URL` to set.
However, there is an issue for us, some notifications like finding open/close still concatenate `http://localhost:8080` at the beginning: `http://localhost:8080http://myserver/finding/42`

**Reply by alexr3** [reply]
@Juu thanks!
